desktop/apps/editorial_features/components/gucci/templates/head.jade

```diff
@@ -0,0 +1,32 @@
+if sd.IS_NESTED_PATH
+  - page = curation.get('sections')[sd.VIDEO_INDEX]
```

Comment:
Also stopped setting a video index unless there is a slug, so you could also switch to `if sd.VIDEO_INDEX` . -- not blocking but FYI

Reply:
Ok good to know - since it's not part of the code now, let's leave that until after you've added that in.

Reply:
Oh actually I have a use for this var, so let's leave it!